canonical/ubuntu.com
# WD-2765 Build the 18-04 out of service page

## Done

- Build the 18-04 out-of-service page from the [copy doc](https://docs.google.com/document/d/1gbygbkkqvWJ-PbkOfTpeL-n8sKlgY-88F_4CNbIPqJA/edit#)

## QA

- Open the demo and go to /18-04
- Check it matches the design: https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?node-id=3628-18766&t=1PbS0G37UUwWbRvh-0
- Check the content matches the copy doc above
- Check different screen sizes
